docs/en/core/comments

```diff
@@ -0,0 +1,60 @@
+# Working With Comments
+
+When writing or editing your work — or that of others — you may well wish to leave comments in a document. To leave a comment in markdown, precede the text of your comment with `<!--` and end it with `-->`. If you're familiar with HTML, you'll notice that this is the same markup used for HTML comments — as a rule, markdown supports HTML syntax.
+
+For a single-line comment, the comment markup can all go on the same line as the comment itself:
+
+```html
+<!-- This is a single-line comment -->
+```
+
+For a comment that spans multiple lines, it's simplest to place the opening comment markup and the closing markup on their own lines, sandwiching the text of your comment in between:
+
+```html
+<!--
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+
+Nunc et lobortis enim. Suspendisse ac sagittis lectus, eget consequat odio. Aliquam congue molestie justo in ullamcorper.
+
+Nulla blandit dignissim erat, sit amet luctus odio luctus et. Etiam euismod libero et lectus finibus, ac euismod metus iaculis.
+-->
+```
+
+To make inserting comments dead simple, Zettlr includes a comment keyboard shortcut. Typing `Ctrl-Shift-c` will auto-complete the basic comment syntax, placing your cursor in the center, between the opening and closing angle brackets, so you can begin typing right away:
```

Comment:
I think I've switched to `<kbd>` for keyboard shortcuts – also don't forget us Mac users!

Reply:
Done!